index.html

```diff
@@ -2,28 +2,29 @@
 <html>
   <head>
     <title>Chat Room</title>
+    <link rel="stylesheet" href="style-file.css">
   </head>
 
   <body>
-    <header>
+    <header class="header-style">
```

Comment:
Remove the "-style" notation from all the class names in this file. It is just repeated text. And we already know that classes are used for adding styles